public-html/login.php

```diff
@@ -60,15 +63,20 @@
 				if ($row === false) {
 					$_SESSION["error"] = "Incorrect password.";
 					header("LOCATION: login.php");
-				} else {
-					$_SESSION["uname"] = $row['uname'];
-					$_SESSION["email"] = $_POST['email'];
-					$_SESSION["admin"] = $k->admin($_SESSION["email"]);
-					$_SESSION["uno"] = $k->uno($_SESSION["email"]);
-					$_SESSION["success"] = "Login success";
-					$_SESSION["logged_in"] = "pass";
-					if (!empty($_POST['remember'])) {
-						setcookie("email", $_POST['email'], time() + (10 * 365 * 24 * 60 * 60));
+				}
+				else
+				{
+					$_SESSION["uname"]=$row['uname'];
+					$_SESSION["email"]=$_POST['email'];
+					$_SESSION["admin"]=$k->admin($_SESSION["email"]);
+					$_SESSION["uno"]=$k->uno($_SESSION["email"]);
+					$ob=new book;
+					$ob->issue_check($_SESSION["uno"]);
+					$_SESSION["success"]="Login success";
+					$_SESSION["logged_in"]="pass";
+					if (!empty($_POST['remember']))
+					{
+						setcookie ("email", $_POST['email'], time()+ (10 * 365 * 24 * 60 * 60));
```

Comment:
Session management should be the part of session class, Please do create a follow up to refactor this.

Reply:
@AJV009